ee/app/livechat-enterprise/server/hooks/saveDefaultAgentAfterTakeInquiry.js

```diff
@@ -0,0 +1,28 @@
+import { callbacks } from '../../../../../app/callbacks';
+import { settings } from '../../../../../app/settings';
```

Comment:
we're already removing the index files from modules root folders, so please always import at least from `server` or `client` folders

```suggestion
import { callbacks } from '../../../../../app/callbacks/server';
import { settings } from '../../../../../app/settings/server';
```